Samples/CustomOperatorCPU/customize_model/scripts/debug_all_outputs.py

```diff
@@ -18,6 +18,10 @@ def create_modified_model(args):
         for output in node.output:
             intermediate_outputs.add(output)
 
+    # remove final graph outputs

```

Comment:
Which spec in onnx are we violating? Can we add a link as a comment here?

Reply:
The final graph outputs are already available as the results of the evaluation. Turns out that the onnx specification defines the execution of a graph to be finished once all the graph outputs are completed. Therefore inserting debug operators after the graph outputs is introducing undefined behavior. And doesn't add anything when it comes to debugging. I would say these are invalid models.

However, onnxruntime should fail gracefully when running this kind of model, when in reality it works some of the time and then happened to crash when running on a model with FP16.

Either way, my helper script shouldn't output this extra debug operator either way, and that is what this pull request addresses.

And yes I will add a link in a comment

Reply:
ahh... this answers my question below.  thx !